tests/topotests/lib/topotest.py

```diff
@@ -2001,7 +2017,7 @@ def start_daemon(daemon, extra_opts=None):
 
         # Start Zebra after mgmtd
         if "zebra" in daemons_list:
-            start_daemon("zebra", "-s 90000000")
+            start_daemon("zebra", "-s 90000000 -M fpm")
```

Comment:
no.  We will not start zebra with the fpm module for every single test.  You need to abstract the ability to pass in the appropriate module ( I think this is already done in other places.  See how the snmp module is started